static/app/data/forms/userFeedback.tsx

```diff
@@ -28,7 +28,7 @@ const formGroups: JsonFormObject[] = [
         label: t('Enable Crash Report Notifications'),
         help: () =>
           tct(
-            'Get notified on [crashReportModalDocsLink: Crash Report Modal and User Report API submissions]. [feedbackWidgetDocsLink: Feedback widget] notifications are not affected by this setting and are on by default.',
+            'Get notified on [crashReportModalDocsLink: Crash Report Modal] and User Feedback API (web endpoint and pre-SDK v8) submissions. [feedbackWidgetDocsLink: Feedback widget] notifications are not affected by this setting and are on by default.',
```

Comment:
Could we add a `webApiEndpointLink` also?

```suggestion
            'Get notified on [crashReportModalDocsLink: Crash Report Modal] and User Feedback API ([webApiEndpointLink:web endpoint] and pre-JS SDK v8) submissions. [feedbackWidgetDocsLink: Feedback widget] notifications are not affected by this setting and are on by default.',
```